VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionPostsSearchFragment.java

```diff
@@ -22,8 +24,11 @@
 
     @InjectExtra(value = Router.EXTRA_SEARCH_QUERY, optional = true)
     private String searchQuery;
+
     private SearchThreadListTask searchThreadListTask;
 
+    private int nextPage = 1;
```

Comment:
No, It might seem like "page" is the current page number. But in fact it is the next (not yet fetched) page number.